keps/sig-api-machinery/00xx-admission-webhooks-to-ga.md

```diff
@@ -130,27 +132,36 @@ type Webhook struct {
 ### Scope
 
 Current webhook Rules applies to objects of all scopes. That means a Rule can use wildcards 
-to target both namespaced and cluster scoped objects. The proposal is to add a scope field 
+to target both namespaced and cluster-scoped objects. The proposal is to add a scope field 
 to Admission Webhook configuration to limit webhook target on namespaced object or cluster 
-scoped objects. This enables webhook developers to target all namespace objects or all 
-cluster-scoped objects. The field will be added to both v1 and v1beta1. The field is optional 
-and empty value means no scope restriction.
+scoped objects. This enables webhook developers to target all namespaced objects or all 
+cluster-scoped objects. Namespace objects themselves are considered cluster-scoped.
+The field will be added to both v1 and v1beta1.
+The field is optional and defaults to "*", meaning no scope restriction.
 
 ```golang
 type ScopeType string
 
 const (
-     // ClusterScope means that scope is limited to cluster objects.
+     // ClusterScope means that scope is limited to cluster-scoped objects.
+     // Namespace objects are considered cluster-scoped.
```

Comment:
"are considered" makes it sound like we've chosen arbitrarily.

`// Kubernetes has a namespace structure that is flat (no nesting); for this reason, namespace objects themselves are cluster scoped.`

Reply:
(multiple places)

Reply:
changed to a statement of fact